src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -23,7 +23,7 @@ namespace APIViewWeb.Repositories
 {
     public class PullRequestManager
     {
-        static readonly string REVIEW_DIFF_URL = "https://apiview.dev/Assemblies/Review/{ReviewId}?diffOnly=True&diffRevisionId={NewRevision}";
+        static readonly string REVIEW_DIFF_URL = "{ReviewId}?diffOnly=True&diffRevisionId={NewRevision}";
```

Comment:
nit, since this is no longer an URL, should we rename it to e.g. REVIEW_DIFF_PARAMETER (actually it is path parameter + query parameters...)?

Or it stays as an URL with host as parameter, as Josh suggested.

Reply:
Not required. I have made the changes as per comment from Josh.